src/main.ts

```diff
@@ -29,31 +34,48 @@ Sentry.init({
   debug: config.sentry.environment == 'development',
 });
 
-// Start BatchDelete queue polling
-new BatchDeleteHandler(new EventEmitter());
+(async () => {
```

Comment:
Setup of the express server should probably also be refactored into `./server`, maybe along with a corresponding `getTestServer` function if we want to send requests through express during testing.  A smaller stopgap alternative is included in the test proposals.